index.js

```diff
@@ -1,6 +1,8 @@
-'use strict';
-const fs = require('fs');
-const path = require('path');
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
```

Comment:
+1 for using [`node:`-prefixed imports](https://2ality.com/2021/12/node-protocol-imports.html)